.bootstrap/.core/architecture/features/workspace.md

```
@@ -0,0 +1,564 @@
+- [1. Feature Context](#1-feature-context)
+  - [1. Overview](#1-overview)
+  - [2. Purpose](#2-purpose)
+  - [3. Actors](#3-actors)
+  - [4. References](#4-references)
+  - [5. Non-Applicability](#5-non-applicability)
+  - [6. Out of Scope](#6-out-of-scope)
+- [2. Actor Flows (CDSL)](#2-actor-flows-cdsl)
+  - [Workspace Initialization](#workspace-initialization)
+  - [Workspace Source Management](#workspace-source-management)
+  - [Workspace Info Display](#workspace-info-display)
+- [3. Processes / Business Logic (CDSL)](#3-processes-business-logic-cdsl)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)